Functions.Templates/Templates/DurableFunctionsEntityHttpStart-TypeScript/index.ts

```diff
@@ -0,0 +1,19 @@
+﻿﻿import { AzureFunction, Context, HttpRequest } from "@azure/functions"
+import * as df from "durable-functions"
+
+const httpStart: AzureFunction = async function (context: Context, req: HttpRequest) {
+    const client = df.getClient(context);
+    const id: string = context.bindingData.id;
+    const entityId = new df.EntityId("Counter", id);
+
+    // reset value
+    await client.signalEntity(entityId, "reset", 1);
+
+    // increment value
+    await client.signalEntity(entityId, "add", 1);
+
+    // reads current state of entity
+    return await client.readEntityState<number>(entityId);
```

Comment:
Wonder if it's better to leave these out, as it might lead someone to believe that these execute synchronously and the `readEntityState` reflects the result of the previous `signalEntity` calls.

Reply:
What would you recommend adding in place of these calls? Would it be better to just have one call to add to the `Counter` entity?

Reply:
Actually, now that I think about it, can we do something like this?

```typescript
import { AzureFunction, Context, HttpRequest } from "@azure/functions"
import * as df from "durable-functions"

const httpStart: AzureFunction = async function (context: Context, req: HttpRequest) {
    const client = df.getClient(context);
    const id: string = context.bindingData.id;
    const entityId = new df.EntityId("Counter", id);

    if (req.method === "POST") {
         // increment value
        await client.signalEntity(entityId, "add", 1);
    } else {
        // read current state of entity
        return await client.readEntityState<number>(entityId);
    }
};

export default httpStart;
```

This should work quite well for a quickstart. @bachuv @cgillum any thoughts on this? If we're all in agreement, the JS and C# templates can do the same. Thanks.

Reply:
I like it! It is simple and avoids the problem you called out earlier.

Reply:
Looks good to me! I'll make the changes for JS and C# too.